### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#387]
# Related Addresses

```
0x597ddaf79527d46a3d838ed4b06bb37ebba8fc87
0x53cfbfe27706bb10f765e7ef64912533e215d724
0x226d0832f19396d747ba74bef1436f14171c5e2d
0x9b98a74aaa8ef218bd3f69c46d9eba831e444804
0x2d85c30b8684669420996415efc0c1dfd87d79d5
0x3fb382e6ca785857c907ac0f07a0361ff1f4d8b9
0x7e49ce9ae98313e8b55cc58779083a47462139fe
0x98381d34fe2a7ef6d8ddfa4024c5aaed836144d0
0x2b45ba3a51c0ecd4504845bf1cababd5f362121b
0x2b7d76047b7c452df3894de7ec1816613e6d6a0b
0x01ef4da8a54072ec387ec227a54a1ac9316252f4
0x074650d0e4957fbaa40fb15cca4407878100d979
```

# Reasoning

All addresses receive funds (on Polygon) from the previous address on the chain, they interact with the same (or similar) set of contracts (e.g. they all do a 1100-1220 hUSDT swapAndSend between Sep 17 and Sep 18, 2021) and then transfer funds to the next address on the chain.

# Methodology

Same as Issue #354

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy . We have verified that the addresses in this report are Sybil attackers.

The report included 12 eligible addresses as Sybil attackers which means you are eligible for 3614.7103193516143855 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).


The qualified addresses are as follows:

```
0x597ddaf79527d46a3d838ed4b06bb37ebba8fc87
0x53cfbfe27706bb10f765e7ef64912533e215d724
0x226d0832f19396d747ba74bef1436f14171c5e2d
0x9b98a74aaa8ef218bd3f69c46d9eba831e444804
0x2d85c30b8684669420996415efc0c1dfd87d79d5
0x3fb382e6ca785857c907ac0f07a0361ff1f4d8b9
0x7e49ce9ae98313e8b55cc58779083a47462139fe
0x98381d34fe2a7ef6d8ddfa4024c5aaed836144d0
0x2b45ba3a51c0ecd4504845bf1cababd5f362121b
0x2b7d76047b7c452df3894de7ec1816613e6d6a0b
0x01ef4da8a54072ec387ec227a54a1ac9316252f4
0x074650d0e4957fbaa40fb15cca4407878100d979
```

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.  
0xD47CB1835EB36e4A108C33E446449192426Fc160 This is a brand new empty address, why can't he provide the address he interacts with, because he is the real empty hunter. He circumvented it with better technical isolation. So here's my reasoning, I want 25% of his reward to go to me and then the remaining 75% to all eligible real users. Just like our rules.
@shanefontaine
